Assets/Scripts/UI/PlayerStatUI.cs

```diff
@@ -86,7 +70,7 @@ void OnDestroy()
 
     private void OnInventoryChange(Item item)
     {
-        SetBaseGunStats(GunFactory.GetGunStats(playerIdentity.Body, playerIdentity.Barrel, playerIdentity.Extension));
+        SetBaseGunStats(GunFactory.GetGunStats(StaticInfo.Singleton.StartingBody, StaticInfo.Singleton.StartingBarrel, StaticInfo.Singleton.StartingExtension));
     }
 
     private void OnBiddingPlatformChange(BiddingPlatform platform)
```

Comment:
Hm, we don't really need this one anymore, right?

Reply:
Nope, I can remove the depreceated code 👍